cookbook/security/remember_me.rst

```diff
@@ -171,21 +171,25 @@ In the following example, the action is only allowed if the user has the
         // ...
     }
 
-You can also choose to install and use the optional JMSSecurityExtraBundle_,
-which can secure your controller using annotations:
+If your application is based on the Symfony Standard Edition, you can also secure
+your controller using annotations:
 
 .. code-block:: php
 
-    use JMS\SecurityExtraBundle\Annotation\Secure;
+    use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
 
     /**
-     * @Secure(roles="IS_AUTHENTICATED_FULLY")
+     * @Security("has_role('IS_AUTHENTICATED_FULLY')")
      */
     public function editAction($name)
     {
         // ...
     }
 
+.. versionadded:: 2.4
+    The ``@Security`` annotation was introduced in SensioFrameworkExtraBundle 3.0,
+    which can only be used with Symfony 2.4 or later.
```

Comment:
I copied this note from this other note:

```
The @Security annotation was introduced in SensioFrameworkExtraBundle 3.0. This bundle 
version can only be used with Symfony 2.4 or later (see the SensioFrameworkExtraBundle 
release cycle).
```

Source: http://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/annotations/security.html

Should we also fix the original note?